contents/code/tiling.js

```diff
@@ -256,7 +256,11 @@ Tiling.prototype.resizeTile = function(tile){
         if (tile != null) {
             var tileIndex = tile.tileIndex;
             var client = tile.clients[0];
-            this.layout.resizeTile(tileIndex, client.geometry);
+            var geometry = Qt.rect(client.geometry.x - this.windowsGapSizeWidth,
+                client.geometry.y - this.windowsGapSizeHeight,
+                client.geometry.width + this.windowsGapSizeHeight * 2,
```

Comment:
This should be Width, not Height.

Reply:
Oh, right. Sorry, I should've checked it better. 
There's also the same issue at line 238 (since I copy-pasted the code from there): https://github.com/kwin-scripts/kwin-tiling/blob/8a5356860f55050ff51fc9b2c176d78f26000b99/contents/code/tiling.js#L236-L240
Should I fix it with this commit/second commit or ignore it for now?

Reply:
I think this can still be tested with the mistake, but so far I've not had the time for that.

Reply:
Sure, no problem :smiley: